tests/test_transforms/test_math/test_binary_operator_transform.py

```diff
@@ -0,0 +1,339 @@
+    "operand, left_column, right_column, out_column",
+    [
+        ("+", "feature", "target", "target"),
+        ("-", "feature", "target", "target"),
+        ("*", "feature", "target", "target"),
+        ("/", "feature", "target", "target"),
+        ("/=", "feature", "target", "target"),
+        ("%", "feature", "target", "target"),
+        ("**", "feature", "target", "target"),
+        ("==", "feature", "target", "target"),
+        (">=", "feature", "target", "target"),
+        ("<=", "feature", "target", "target"),
+        (">", "feature", "target", "target"),
+        ("<", "feature", "target", "target"),
+        ("+", "target", "feature", "target"),
+        ("-", "target", "feature", "target"),
+        ("*", "target", "feature", "target"),
+        ("/", "target", "feature", "target"),
+        ("/=", "target", "feature", "target"),
+        ("%", "target", "feature", "target"),
+        ("**", "target", "feature", "target"),
+        ("==", "target", "feature", "target"),
+        (">=", "target", "feature", "target"),
+        ("<=", "target", "feature", "target"),
+        (">", "target", "feature", "target"),
+        ("<", "target", "feature", "target"),
+        ("+", "feature", "target", "feature"),
+        ("-", "feature", "target", "feature"),
+        ("*", "feature", "target", "feature"),
+        ("/", "feature", "target", "feature"),
+        ("/=", "feature", "target", "feature"),
+        ("%", "feature", "target", "feature"),
+        ("**", "feature", "target", "feature"),
+        ("==", "feature", "target", "feature"),
+        (">=", "feature", "target", "feature"),
+        ("<=", "feature", "target", "feature"),
+        (">", "feature", "target", "feature"),
+        ("<", "feature", "target", "feature"),
+        ("+", "target", "feature", "feature"),
+        ("-", "target", "feature", "feature"),
+        ("*", "target", "feature", "feature"),
+        ("/", "target", "feature", "feature"),
+        ("/=", "target", "feature", "feature"),
+        ("%", "target", "feature", "feature"),
+        ("**", "target", "feature", "feature"),
+        ("==", "target", "feature", "feature"),
+        (">=", "target", "feature", "feature"),
+        ("<=", "target", "feature", "feature"),
+        (">", "target", "feature", "feature"),
+        ("<", "target", "feature", "feature"),
+        ("+", "feature", "target", "new_col"),
+        ("-", "feature", "target", "new_col"),
+        ("*", "feature", "target", "new_col"),
+        ("/", "feature", "target", "new_col"),
+        ("/=", "feature", "target", "new_col"),
+        ("%", "feature", "target", "new_col"),
+        ("**", "feature", "target", "new_col"),
+        ("==", "feature", "target", "new_col"),
+        (">=", "feature", "target", "new_col"),
+        ("<=", "feature", "target", "new_col"),
+        (">", "feature", "target", "new_col"),
+        ("<", "feature", "target", "new_col"),
+        ("+", "target", "feature", "new_col"),
```

Comment:
Same about this.

Reply:
a = operand(a,b),  and a = operand(b,a) required different logic in inverese transform so I decided to add this tests in common tests in order to check all possible problems